lib/broccoli/ember-app.js

```diff
@@ -1845,6 +1845,30 @@ function addOutputFile(container, assetPath, options) {
     container[outputFile] = [];
   }
 
+  if (~container[outputFile].indexOf(assetPath)) {
```

Comment:
My issue with it isn't just that it is a bitwise operation, it is that JavaScript's coercion is allowed to once again run wild.

In general, we encourage `===` over `==` and my feeling is that discouraging `~` is an extension of that.

My brain parses:
```js
if (~x.indexOf(y)) {

}
```

as:

```js
if (toBoolean(-(toNumber(x.indexOf(y)) + 1)) {

}
```
whereas

```